sdks/java/core/build.gradle

```diff
@@ -98,7 +98,7 @@ dependencies {
   permitUnusedDeclared enforcedPlatform(library.java.google_cloud_platform_libraries_bom)
   provided library.java.json_org
   implementation library.java.everit_json_schema
-  implementation library.java.snake_yaml
+  shadow library.java.snake_yaml
```

Comment:
I'm afraid of this getting changed again. Is it necessarily part of this change? Is it safe now?